### PR TITLE
Completed data-structure-stacks - Ellie Hwang

### PR DESCRIPTION
**Pull request by ellie-hwang**
data-structures-stacks-01

### CODE REVIEW

File: data-structures-stacks/max-value.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/* exported maxValue */
+
+function maxValue(stack) {
+  const stackValues = [];
+  let max = -Infinity;
```
Comment (thebearingedge) [inline data-structures-stacks/max-value.js:5]: Please solve this without an extra array.

File: data-structures-stacks/get-2nd-from-top.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+/* exported get2ndFromTop */
+
+function get2ndFromTop(stack) {
+  const stackValues = [];
+  while (stack.peek() !== undefined) {
```
Comment (thebearingedge) [inline data-structures-stacks/get-2nd-from-top.js:5]: Please solve this without an extra array.

File: data-structures-stacks/insert-under-top.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+/* exported insertUnderTop */
+
+function insertUnderTop(stack, value) {
+  if (stack.peek() === undefined) {
+    return;
+  }
+  const stackValues = [];
+  while (stack.peek() !== undefined) {
```
Comment (thebearingedge) [inline data-structures-stacks/insert-under-top.js:8]: Please solve this without an extra array.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: data-structures-stacks/insert-under-top.js (hunk 3 above)
Comment (ellie-hwang) [reply data-structures-stacks/insert-under-top.js]: Gotcha, arrays have been removed!

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**